components/components/navigation/menu-main/menu-main.css

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file
+ * This file is used to style the main menu.
+ */
+
+/*
+  Main Menu
+
+  This section presents the Main Menu of the website.
+
+  Markup: menu-main.html.twig
+
+  Style guide: components.navigation.main_menu
+*/
+
+.menu-main__toggle {
+  width: 40px;
+  height: 40px;
+  line-height: 1;
+  padding: 0;
+  border: none;
+  text-align: left;
+  background-color: transparent;
+}
+.menu-main {
```

Comment:
I usually try to always place the main element as first in the CSS

Reply:
OK